src/object_info.rs

```diff
@@ -151,21 +151,16 @@ impl ObjectInfo {
         }
 
         let stack = get_stack_info(Some(main_object), pe_object);
-        let symbols = match platform {
-            Platform::Linux | Platform::Mac => super::symbol::add_executable_section_symbols(
-                collector.syms,
-                main_file_name,
-                main_object,
-            ),
-            Platform::Win => super::symbol::append_dummy_symbol(
-                collector.syms,
-                pe_file_name.unwrap_or(main_file_name),
-            ),
-        };
+        let symbols = super::symbol::add_executable_section_symbols(
+            collector.syms,
+            main_file_name,
+            main_object,
+        );
 
         let file_name = match (&main_object, &pe_file_name) {
             (Object::Elf(elf), _) => elf.name().unwrap_or(main_file_name),
             (Object::MachO(macho), _) => macho.name().unwrap_or(main_file_name),
+            (Object::Pe(_pe), _) => pe_file_name.unwrap_or(main_file_name),
```

Comment:
The problem with the `mozwer` test is here. This overrides the `main_file_name` which in this case should be the debug file name (i.e. `mozwer.pdb`) with the PE name (`mozwer.dll`). I think the proper fix here is to drop this line and adjust [this](https://github.com/mozilla/dump_syms/blob/0bd3a3daf7f7119e0861657083d3233172d5263e/src/windows/pdb.rs#L38) instead so that we use the PE name when the PDB name is not available:
```rust
        let pdb_name = pe.debug_file_name().unwrap_or(pe_name.into()).to_string();
```

Reply:
Sounds good, implemented.